charts/devices-api/values.yaml

```diff
@@ -67,15 +67,16 @@ env:
   DEVICE_DEFINITION_METADATA_TOPIC: table.device.definition.metadata
   BLACKBOOK_API_URL: https://service.blackbookcloud.com/UsedCarWS/UsedCarWS
   ELASTIC_DEVICE_STATUS_INDEX: device-status-dev-*
-  DIMO_REGISTRY_ADDR: '0x4De1bCf2B7E851E31216fC07989caA902A604784'
+  DIMO_REGISTRY_ADDR: "0x4De1bCf2B7E851E31216fC07989caA902A604784"
   MONITORING_SERVER_PORT: 8888
   ENABLE_PRIVILEGES: true
   TOKEN_EXCHANGE_JWK_KEY_SET_URL: http://dex-roles-rights.dev.svc.cluster.local:5556/keys
   CONTRACT_EVENT_TOPIC: topic.contract.event
-  VEHICLE_NFT_ADDRESS: '0x90c4d6113ec88dd4bdf12f26db2b3998fd13a144'
+  VEHICLE_NFT_ADDRESS: "0x90c4d6113ec88dd4bdf12f26db2b3998fd13a144"
   AUTOPI_NFT_IMAGE: autopi.png
   VINCARIO_API_URL: https://api.vindecoder.eu/3.2
   CHATGPT_URL: https://api.openai.com/v1/chat/completions
+  DEVICE_AFTERMARKET_CONTRACT_ADDRESS: "0x5D666F215a85B87Cb042D59662A7ecd2C8Cc44e6"
```

Comment:
What's this address? I think on dev it should be `0xcf9af64522162da85164a714c23a7705e6e466b3`, and `0x9c94c395cbcbde662235e0a9d3bb87ad708561ba` on prod.